marlowe-playground-client/src/Simulation/BottomPanel.purs

```diff
@@ -156,7 +157,7 @@ panelContents state CurrentStateView =
     else
       (headerRow "Warnings" ("type" /\ "details" /\ mempty /\ mempty /\ mempty)) <> foldMap displayWarning' warnings
 
-  error = previewOn state (_marloweState <<< _Head <<< _executionState <<< _Just <<< _transactionError)
+  error = (previewOn state (_marloweState <<< _Head <<< _executionState <<< _Just <<< _transactionError)) >>= (\x -> x)
```

Comment:
you can use `previewOn state (_marloweState <<< _Head <<< _executionState <<< _Just <<< _transactionError <<< _Just)` here instead of the `>>=` stuff